src/render/SDL_sysrender.h

```diff
@@ -148,6 +148,8 @@ struct SDL_Renderer
                             const SDL_Rect * rect,
                             const Uint8 *Yplane, int Ypitch,
                             const Uint8 *UVplane, int UVpitch);
+    int (*UpdateTextureOES) (SDL_Renderer * renderer, SDL_Texture * texture,
```

Comment:
Why is this guarded by `SDL_HAVE_YUV` ?

Reply:
My original goal is show yuv frame directly, so I make it guarded by SDL_HAVE_YUV.
Perhaps it is inappropriate
I'll fix it

Reply:
fixed